blueprints/datamodel.tex

```diff
@@ -355,10 +465,33 @@ \subsubsection{Status}
 	Servers.
 \end{itemize}
 
+\paragraph{Tenant}
+
+\paragraph{Vanity Hostnames}
```

Comment:
If this is to replace HOST_REGEXs, what do you intend to do about the other kinds of regexes?

Reply:
Nothing, yet. Still has to be discussed because I don't understand the full use-case set for custom match lists.